x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -16,10 +16,10 @@ import { ElasticsearchClient } from 'kibana/server';
 import {
   ActionGroup,
   ActionGroupIdsOf,
-  AlertInstance,
-  AlertInstanceContext,
-  AlertInstanceState,
-  AlertTypeState,
+  AlertInstance as Alert,
+  AlertInstanceContext as AlertContext,
+  AlertInstanceState as AlertState,
+  AlertTypeState as RuleTypeState,
 } from '../../../../../alerting/server';
 import {
   AlertParams,
```

Comment:
@ersin-erdal Shall we rename this to `RuleParams`? Although so far I had the impression we have RuleTypeParams, so I am a bit confused here.